topdown/encoding.go

```diff
@@ -50,6 +50,18 @@ func builtinJSONUnmarshal(a ast.Value) (ast.Value, error) {
 	return ast.InterfaceToValue(x)
 }
 
+func builtinJSONIsValid(a ast.Value) (ast.Value, error) {
+
+	str, err := builtins.StringOperand(a, 1)
+	if err != nil {
+		return nil, err
+	}
+
+	var x interface{}
+	err = util.UnmarshalJSON([]byte(str), &x)
```

Comment:
[nit] We don't care for the result, so we could also do something like this, I think:
```suggestion
	err = util.UnmarshalJSON([]byte(str), &struct{}{})
```
(I've had to try myself, https://play.golang.org/p/Me2tQPEP7bU)

Reply:
@jaspervdj-luminal ⚠️ [I was wrong!](https://play.golang.org/p/-EPAr5cqRdn) Sorry for putting you off the right track. `interface{}` is the proper thing to use.